app/src/componentsV2/Tabs/store/tabServiceStore.ts

```diff
@@ -229,11 +237,3 @@ export const useTabServiceWithSelector = <T>(selector: (state: TabServiceState)
   const store = useContext(TabServiceStoreContext);
   return useStore(store, useShallow(selector));
 };
-
-/**
- * Usage: const openTab = useTabServiceStore().use.openTab()
- */
-export const useTabServiceStore = () => {
-  const store = useContext(TabServiceStoreContext);
-  return store;
-};
```

Comment:
Removing this because it was not being consumed as a hook. It was not reactive to the changes in state